package.json

```diff
@@ -17,7 +17,7 @@
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-router-dom": "5.2.0",
-    "react-scripts": "3.4.1"
+    "react-scripts": "^3.4.1"
```

Comment:
```suggestion
    "react-scripts": "3.4.1"
```

Reply:
The dependencies are pinned in this project. I will update them manually on the next release